doc/content/guides/connecting-gateways/kerlinkwirnet/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Kerlink Wirnet Station"
+description: ""
+weight: 1
+---
+
+Kerlink Wirnet Station is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.kerlink.com/product/wirnet-station/). This page guides you to connect it to The Things Stack.
+
+## Prerequisites
+
+1. User account on The Things Stack with rights to create Gateways and API Keys.
+2. Kerlink Wirnet Station:
+  1. For configuration via provisioning script: Common Packet Forwarder installed and enabled and/or gateway running firmware version `3.0` or higher.
+  2. For manual configuration: Common Packet Forwarder installed and enabled
+
+> NOTE: Minimum CPF version tested is `v1.1.6`
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}). Choose a **Gateway ID** that is at least 6 characters in length. An **EUI** is not necessary.
+
+Create an API Key with Gateway Link rights for this gateway using the same instructions. Copy the key and save it for later use.
+
+## Configuration
+
+All further steps will assume the gateway is available at `192.168.4.155` and the stack address is `https://thethings.example.com`, please replace these by the addresses appropriate for your setup.
+
+### Provisioning script
+
+Please refer to https://github.com/TheThingsNetwork/kerlink-station-firmware#provisioning for detailed up-to-date documentation.
+
+1. Ensure your public key is authorized for `root` user on the gateway. If that's not the case (you need to enter `root` password to login into the gateway via SSH) - execute `ssh-copy-id root@192.168.4.155`.
+2. Execute `curl -sL https://raw.githubusercontent.com/TheThingsNetwork/kerlink-station-firmware/master/provision.sh | bash -s -- "192.168.4.155" "https://thethings.example.com"`
+3. In case the CPF was installed in process of provisioning, you may need to configure `lorad` for your setup, disable any active packet forwarders and enable CPF. Please refer to ["lorad configuration"](http://wikikerlink.fr/lora-station/doku.php?id=wiki:cpf#lorad_configuration) and ["Starting and stopping the packet forwarder"](http://wikikerlink.fr/lora-station/doku.php?id=wiki:cpf#starting_and_stopping_the_packet_forwarder) sections of [CPF documentation](wikikerlink.fr/lora-station/doku.php?id=wiki:cpf) on [Kerlink Wiki](http://wikikerlink.fr/lora-station/doku.php).
```

Comment:
1. What does "you may need to configure" mean exactly?
2. Kerlink Wiki is behind a login; for 80% of the users this leads to a dead end

Reply:
I added setting the frequency plan and restarting the packet forwarder to provisioning script in https://github.com/TheThingsNetwork/kerlink-station-firmware/pull/9. I think this configuration should be enough for most users. More advanced configuration does not belong here anyway and will most probably require access to the wiki to be done right, unless we mirror the whole wiki, which I don't think we should do.